src/goom_config.h

```diff
@@ -2,11 +2,11 @@
 #define GOOM_CONFIG
 
 #ifdef MID
-#define GOOM_TEXTURE_WIDTH  256
-#define GOOM_TEXTURE_HEIGHT 256
-#else  // MID
 #define GOOM_TEXTURE_WIDTH  512
-#define GOOM_TEXTURE_HEIGHT 512
+#define GOOM_TEXTURE_HEIGHT 256
+#else // MID
+#define GOOM_TEXTURE_WIDTH  1280
+#define GOOM_TEXTURE_HEIGHT 720
 #endif // MID
```

Comment:
goom won't say anything about this. this is in the presentation layer - opengl stuff.